python/ray/_private/worker.py

```diff
@@ -1355,6 +1355,14 @@ def init(
         else:
             usage_lib.set_usage_stats_enabled_via_env_var(False)
 
+        # assign a default storage if there is no storage.
```

Comment:
I'm wondering whether we also need to print any logs for this.

Reply:
We can't print any log spam on ray init.